tests/common/nebula_test_suite.py

```diff
@@ -351,18 +352,97 @@ def to_value(self, col):
         elif isinstance(col, str):
             value.set_sVal(col.encode('utf-8'))
         elif isinstance(col, dict):
-            map_val = CommonTtypes.Map()
-            map_val.kvs = dict()
-            for key in col:
-                ok, temp = self.to_value(col[key])
+            if "type" not in col:
```

Comment:
Maybe we need to modify the type to `__TYPE__`. When the result is map type and has the key is `type`.

Reply:
done